components/esp_lvgl_port/CMakeLists.txt

```diff
@@ -3,4 +3,7 @@ idf_component_register(SRCS "esp_lvgl_port.c" INCLUDE_DIRS "include" REQUIRES "e
 idf_build_get_property(build_components BUILD_COMPONENTS)
 if("espressif__button" IN_LIST build_components)
     target_link_libraries(${COMPONENT_LIB} PRIVATE idf::espressif__button)
+endif()
+if("espressif__esp_lcd_touch" IN_LIST build_components)
```

Comment:
One last point: the touch component will be named `espressif__esp_lcd_touch` in case it is coming from IDF component manager. But it will be name `esp_lcd_touch` when using it from local path (ie for debugging purposes).

Could you please check for both names in this condition? and do the same update for `espressif__button`?

Reply:
Done.